k8s/prod/common/reform-scan/notification-service.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: reform-scan-notification-service
+  namespace: reform-scan
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: reform-scan-notification-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: reform-scan-notification-service
+    version: 0.0.11
```

Comment:
aat points to v12 in same (this) pr

Reply:
we do that with separate pr as this would need devops approve. after v12 approved I will bump the other env.